admin/controllers/check-form.js

```diff
@@ -194,47 +194,36 @@ const saveCheckForm = async (req, res, next) => {
   res.redirect('/test-developer/upload-and-view-forms')
 }
 
-/* @TODO: The code below will be refactored in the next PR as part of PBI 17604 */
-// const displayCheckForm = async (req, res, next) => {
-//   req.breadcrumbs('Upload and view forms', '/test-developer/upload-and-view-forms')
-//   res.locals.pageTitle = 'View form'
-//   let canDelete = true
-//   let formData
-//
-//   try {
-//     formData = await checkFormDataService.getActiveFormPlain(req.params.formId)
-//     formData.checkWindows = []
-//     const checkWindows = await checkWindowService.getCheckWindowsAssignedToForms()
-//
-//     if (checkWindows[formData.id]) {
-//       // Form is assigned to one or more check windows
-//       formData.checkWindows = checkWindows[formData.id].map(cw => { return cw.toJSON() })
-//       formData.checkWindows.forEach(cw => {
-//         if (cw.startDate <= Date.now()) {
-//           // we can't delete a form whose check window has started.
-//           canDelete = false
-//         }
-//       })
-//     } else {
-//       formData.checkWindows = []
-//     }
-//
-//     req.breadcrumbs(res.locals.pageTitle)
-//     res.render('test-developer/view-check-form', {
-//       form: formData,
-//       canDelete: canDelete,
-//       breadcrumbs: req.breadcrumbs()
-//     })
-//   } catch (error) {
-//     next(error)
-//   }
-// }
+const displayCheckForm = async (req, res, next) => {
+  req.breadcrumbs('Upload and view forms', '/test-developer/upload-and-view-forms')
+  res.locals.pageTitle = 'View form'
+
+  let formData
+
+  formData = await checkFormDataService.getActiveFormPlain(req.params.formId)
```

Comment:
As this block has two async methods should we wrap it in a try/catch so that if the promise is rejected it won't go unhandled?

Node keeps nagging me that in the future unhandled promises will halt execution.